.github/workflows/nodejs.yml

```diff
@@ -18,3 +18,7 @@ jobs:
       env:
         CI: true
         DATABASE_URL: postgres://alchemy:alchemy@postgres:5432/alchemy
+        TWILIO_ACCOUNT_SID: ${{ secrets.TWILIO_ACCOUNT_SID }}
```

Comment:
nice work, I know that Twilio is a little intimidating, but love that you took the initiative to tackle it.